clang/include/clang/Basic/CodeGenOptions.def

```diff
@@ -366,6 +366,9 @@ VALUE_CODEGENOPT(SSPBufferSize, 32, 0)
 /// The kind of generated debug info.
 ENUM_CODEGENOPT(DebugInfo, codegenoptions::DebugInfoKind, 4, codegenoptions::NoDebugInfo)
 
+/// The kind of generated debug info.
+ENUM_CODEGENOPT(CasFriendliness, codegenoptions::CasFriendlinessKind, 1, codegenoptions::NoCasFriendlyDebugInfo)
```

Comment:
Are we expecting several different modes? Hmm.. as this is in experimental feature, I think I like the idea.

Reply:
Oh, but then in LLVM IR you only reserve a bool for it. I think whatever you do it should be consistent. So maybe just use a flag here too?